tracer.go

```diff
@@ -210,3 +210,16 @@ func (t *tracerImpl) Join(operationName string, format interface{}, carrier inte
 	}
 	return nil, opentracing.ErrUnsupportedFormat
 }
+
+// GetGlobalTracerOptions attempts to get the basictracer.Options
+// struct provided to the global Tracer.  Returns the options and true
+// if the global Tracer is a basictracer implementation; returns
+// false otherwise.
+func GetGlobalTracerOptions() (Options, bool) {
```

Comment:
This is not for regular application code. This is for our implementation to access some of its internal state (an API key) given a Tracer handle, to use for a special-purpose API. Since the Tracer is actually implemented by basictracer, we have no direct access to its state.

It's also not for test code. We know the internal state at the time we construct the Tracer. Getting from Tracer to the internal state is what we're trying to do.